tests/indentblockcommand.js

```diff
@@ -66,6 +85,16 @@ describe( 'IndentBlockCommand', () => {
 				command.execute();
 				sinon.assert.calledTwice( indentBehavior.getNextIndent );
 			} );
+
+			it( 'should be executed only for top-most blocks that can have indentBlock attribute', () => {
+				setData( model,
+					'<paragraph>f[oo</paragraph>' +
+					'<parentBlock><paragraph>foo</paragraph><paragraph>foo</paragraph></parentBlock>' +
+					'<paragraph>f]oo</paragraph>'
+				);
+				command.execute();
+				sinon.assert.calledTwice( indentBehavior.getNextIndent );
```

Comment:
I see the reason why you're relaying on `getNextIndent` calls here instead of checking model.

To have more confidence I'd like to see one test added in `indent`/`using classes` too, where it checks actual output model to make sure that indentation was added where we want it to be.

Reply:
Done.